funding/manager.go

```diff
@@ -1286,6 +1287,54 @@ func (f *Manager) advancePendingChannelState(
 			channel.FundingOutpoint, err)
 	}
 
+	if blockchain.IsCoinBaseTx(confChannel.fundingTx) {
+		// If it's a coinbase transaction, we need to wait for it to
+		// mature. We use 20 blocks over for a margin of safety.
+		const numCoinbaseConfs = 120
+
+		txid := &channel.FundingOutpoint.Hash
+		fundingScript, err := makeFundingScript(channel)
+		if err != nil {
+			log.Errorf("unable to create funding script for "+
+				"ChannelPoint(%v): %v",
+				channel.FundingOutpoint, err,
+			)
+
+			return err
+		}
+
+		confNtfn, err := f.cfg.Notifier.RegisterConfirmationsNtfn(
+			txid, fundingScript, numCoinbaseConfs,
+			channel.BroadcastHeight(),
+		)
+		if err != nil {
+			log.Errorf("Unable to register for confirmation of "+
+				"ChannelPoint(%v): %v",
+				channel.FundingOutpoint, err,
+			)
+
+			return err
+		}
+
+		var ok bool
+
+		select {
+		case _, ok = <-confNtfn.Confirmed:
+			// fallthrough
+
+		case <-f.quit:
+			return ErrFundingManagerShuttingDown
+		}
+
+		if !ok {
+			return fmt.Errorf(
```

Comment:
nit: formatting of call to `Errorf()`, can use compact version, here and above.

Reply:
think I did this